Makefile

```diff
@@ -0,0 +1,21 @@
+ARTIFACT_ID=k8s-apply-lib
+VERSION=0.1.0
+GOTAG?=1.18
+MAKEFILES_VERSION=6.0.1
+.DEFAULT_GOAL:=default
```

Comment:
Coming from building linux tools with `make` where the project should build in the most sensible way, I don't agree with you.

Also I don't recollect a general rule for `help` being the go-to target. I DO see your point though that every Cloudogu project has different needs. And the `help` target is still available. 

I rather prefer to check if the project is generally buildable. Since there is no main code with the project being a library for me the sensible thing to do is to run tests and see if anything weird bubbles up to the surface.

Reply:
This sounds reasonable to me.